src/System.Runtime/tests/System/Runtime/Serialization/FormatterServices.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization;
+using Xunit;
+
+namespace System.Tests
+{
+    public static class FormatterServicesTests
+    {
+    	private static Func<Type, Object> geo = FormatterServices.GetUninitializedObject;
```

Comment:
That said... why use a delegate at all?  Why not just call FormatterServices.GetUnitializedObject in each test?